cuml/forest_inference_demo.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Forest Inference Library\n",
```

Comment:
It looks like we consistently refer to this as `FIL` so maybe we should add `(FIL)` to the end. That way you can shorten some of other places `forest inference library` is used to just `FIL` in the notebook.

Reply:
+1 to this.